MODMARK.md

```diff
@@ -11,36 +10,28 @@ Tags and smart punctuation are also part of the language. Information about thes
 
 ## Paragraphs
 
-Paragraphs are sequences of text separated by one or more empty rows. A single backslash is used at the end of a row to join the following row in the compiled text. Double backslash is used to include a break in the compiled text without creating a new paragraph.
-
-Example 1: simple paragraphs
+Each paragraph consist of one or more lines of text, and ends by one or many blank rows, similar to markdown. Line breaks will be kept and may or may not be rendered by the output format. Many output formats such as HTML, Markdown and LaTeX doesn't render line breaks. If you want to escape the line break to ensure that it is outputted on the same line, you can put a backslash at the end of the line. Here is an example of some paragraphs:
 
     This is the first sentence of the first paragraph.
```

Comment:
Maybe we can explicitly label this example as well to be consistent with the others, or maybe even remove all labels.

Reply:
Fixed!